services/core-web/src/components/common/DocumentColumns.js

```diff
@@ -1,61 +1,52 @@
 import React from "react";
-import moment from "moment";
-import {
-  dateSorter,
-  formatDate,
-  formatDateTime,
-  nullableStringSorter,
-} from "@common/utils/helpers";
 import * as Strings from "@common/constants/strings";
-import DocumentLink from "@/components/common/DocumentLink";
-
-export const documentNameColumn = (documentNameColumnIndex, title = "Name") => {
-  return {
-    title,
-    dataIndex: documentNameColumnIndex,
-    sorter: nullableStringSorter(documentNameColumnIndex),
-    render: (text, record) => (
-      <div key={record.key} title={title}>
-        <DocumentLink
-          documentManagerGuid={record.document_manager_guid}
-          documentName={record?.[documentNameColumnIndex]}
-          truncateDocumentName={false}
-        />
-      </div>
-    ),
-  };
-};
+import { Button, Popconfirm } from "antd";
+import { TRASHCAN } from "@/constants/assets";
+import {
+  renderDateColumn,
+  renderDocumentLinkColumn,
+  renderTextColumn,
+} from "./CoreTableCommonColumns";
 
-export const categoryColumn = (categoryDataIndex, documentCategoryOptionsHash) => {
-  return {
-    title: "Category",
-    dataIndex: categoryDataIndex,
-    render: (text) => <div title="Category">{documentCategoryOptionsHash[text]}</div>,
-  };
+export const documentNameColumn = (
```

Comment:
changed defaults here for data index to match data, and column headers to match Figma. Called other functions for more consistency. Kept them as separate functions here so the headers stay consistent.